css/style.css

```diff
@@ -11,10 +11,55 @@ body {
   background-size: cover;
 }
 
+.grid-container {
+  display: grid;
+  grid-template-columns: auto;
+}
```

Comment:
- [ ] It would be better if we use flexbox instead of the grid we currently have :+1:

Reply:
yes Sir but in the requirements is mandatory to use CSS grid and flexbox together